### PR TITLE
Fix export argument type (T1108890)

### PR DESCRIPTION
**Pull request by GoodDayForSurf**

### CODE REVIEW

File: js/viz/core/base_widget.d.ts [hunk 1]
```diff
@@ -557,7 +558,7 @@ export interface BaseWidgetTooltip {
  * @inherits DOMComponent
  * @namespace DevExpress.viz
  */
-export default class BaseWidget<TProperties> extends DOMComponent<TProperties> {
+export default class BaseWidget<TProperties = Properties> extends DOMComponent<TProperties> {
```
Comment (IlyaKhD) [inline js/viz/core/base_widget.d.ts]: Is this change still required?
`BaseWidget` is not a public type and is used on rare occasions, it could have no default parameter

File: js/viz/export.d.ts [hunk 2]
```diff
@@ -1,11 +1,24 @@
...
+import dxSankey from './sankey';
+import dxTreeMap from './tree_map';
+import dxVectorMap from './vector_map';
+import dxSparkline from './sparkline';
+import dxBullet from './bullet';
+import dxBarGauge from './bar_gauge';
+
+/**
+ * @docid
+ * @public */
```
Comment (IlyaKhD) [inline js/viz/export.d.ts]: Format:
```suggestion
/**
 * @docid
 * @public
 */
```

Comment (IlyaKhD) [reply js/viz/export.d.ts]: About this '@docid' - actually we could remove it. 
It depends whether we want to have a special topic for `VizWidget`, or use a kind of inline description
/cc @vladaskorohodova

Comment (GoodDayForSurf) [reply js/viz/export.d.ts]: without @docid validate-declarations does not pass

File: js/viz/core/base_widget.d.ts [hunk 3]
```diff
@@ -1,5 +1,5 @@
 import DOMComponent, {
-    DOMComponentOptions,
+  DOMComponentOptions,
```
Comment (IlyaKhD) [inline js/viz/core/base_widget.d.ts]: Unnecessary change

**Review by IlyaKhD — DISMISSED** [review]

File: js/viz/export.d.ts [hunk 4]
```diff
@@ -1,11 +1,24 @@
...
 
+import dxChart from './chart';
+import dxPieChart from './pie_chart';
+import dxFunnel from './funnel';
+import dxSankey from './sankey';
+import dxTreeMap from './tree_map';
+import dxVectorMap from './vector_map';
+import dxSparkline from './sparkline';
+import dxBullet from './bullet';
+import dxBarGauge from './bar_gauge';
```
Comment (IlyaKhD) [inline js/viz/export.d.ts]: Use `import type` to remove imports in run-time:

```suggestion
import type dxChart from './chart';
import type dxPieChart from './pie_chart';
import type dxFunnel from './funnel';
import type dxSankey from './sankey';
import type dxTreeMap from './tree_map';
import type dxVectorMap from './vector_map';
import type dxSparkline from './sparkline';
import type dxBullet from './bullet';
import type dxBarGauge from './bar_gauge';
```

**Review by IlyaKhD — APPROVED** [review]
